config/envs/development.py

```diff
@@ -44,13 +31,6 @@ class DevelopmentConfig(Config):
         with open(_test_public_key_path, mode="rb") as public_key_file:
             RSA256_PUBLIC_KEY = public_key_file.read()
 
-    # Redis
-
-    REDIS_INSTANCE_URI = getenv("REDIS_INSTANCE_URI", "redis://localhost:6379")
-    REDIS_MLINKS_URL = f"{REDIS_INSTANCE_URI}/0"
-    REDIS_SESSIONS_URL = f"{REDIS_INSTANCE_URI}/1"
-    SESSION_REDIS = redis.from_url(REDIS_SESSIONS_URL)
```

Comment:
This is the same as defaulted in: https://github.com/communitiesuk/funding-service-design-authenticator/blob/bau-cleanup-config/config/envs/default.py#L252-L255